app/controllers/api/v1/schedules_controller.rb

```diff
@@ -37,12 +37,7 @@ def availableTimeSlots
     api :POST, "/v1/schedules", "Make a booking for the pick up"
     param_group :schedule
     def create
-      @schedule.attributes = schedule_params
-      if @schedule.save
-        render json: @schedule, serializer: serializer, status: 201
-      else
-        render json: @schedule.errors.to_json, status: 422
-      end
+      assign_params_and_render_object(@schedule, serializer, schedule_params)
```

Comment:
Hi @steveyken , appreciate your valuable feedback on this PR.

About this specific method call , I am calling following method for schedules#create
```
def assign_params_and_render_object(data_object, serializer, params={})
       data_object.attributes = params
       if data_object.save
         render json: data_object, serializer: serializer, status: 201
       else
         render json: data_object.errors.to_json, status: 422
       end
     end
```
So it doesn't use cached json and  `data_object = data_object.find( params[:ids].split(",") ) if params[:ids].present?` maybe you were talking about following method call which we are not calling from here

```
    def render_object(data_object, model_name, serializer, params)
      if params[:ids].blank?
         render json: model_name.cached_json
         return
       end
       data_object = data_object.find( params[:ids].split(",") ) if params[:ids].present?
       render json: data_object, each_serializer: serializer
     end
   end
```

So if we call `assign_params_and_render_object` method will it be fine or can we have some undesirable consequences?